Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
 FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
```

Comment:
we should use v1.4 so default encoding is passed to jvm properly

Reply:
done